ember/app/components/nav-bar/template.hbs

```diff
@@ -8,15 +8,17 @@
   </LinkTo>
 
   <div class="uk-navbar-left">
-    <UkTab as |tab|>
-      <tab.link-item @route="cases">
-        <span>
+    <ul class="uk-navbar-nav uk-text-uppercase">
+      <li>
+        <LinkTo @route="cases">
           {{t "nav.documents"}}
-        </span>
-      </tab.link-item>
-      <tab.link-item @route="form-builder">
-        {{t "nav.form-builder"}}
-      </tab.link-item>
-    </UkTab>
+        </LinkTo>
+      </li>
+      <li>
+        <LinkTo @route="form-builder">
+          {{t "nav.form-builder"}}
+        </LinkTo>
+      </li>
+    </ul>
```

Comment:
Why did you refactor that away from `UkTab`? You won't have any active state.

Reply:
Reverted